charts/ontology/templates/deployment.yaml

```diff
@@ -43,17 +46,27 @@ spec:
                 secretKeyRef:
                   name: {{ .Chart.Name }}-secrets
                   key: dsn
-          command: ["java"]
```

Comment:
IMO it's actually better to invoke `java` directly here instead of using Bash :) This way we're not expecting a particular shell to exist on the container image. We should probably update the Cromwell chart to do the same (if it works)

Reply:
Fair point. I can try to get it to work switching back to invoking java directly. May need to play around with how we pass arguments in that case.  I was getting `unsupported argurment` errors when trying to pass the `-javaagent` arg when using the java command directly.

Reply:
I'll play around with it some more to see if I can get it working with the java command pattern.

Reply:
Might be able to pass the `-javaagent` as an env var or something

Reply:
Agree with Chelsea here.